test/nsidc-icesat2/NSIDC-ICESAT2_Regression.ipynb

```diff
@@ -191,10 +191,10 @@
     "        # trajectories. This will be investigated in DAS-2332.\n",
     "        'ATL08': {\n",
     "            'collection_concept_id': Collection(id='C1256432189-NSIDC_CUAT'),\n",
-    "            'granule_id': 'G1261385533-NSIDC_CUAT',\n",
+    "            'granule_id': 'G1260745539-NSIDC_CUAT',\n",
```

Comment:
Another thought, is the comment above this line still true? or should it be removed. @asteiker ?

Reply:
@flamingbear I _think_ this is still true. The ticket (DAS-2332) is still open. But it is not something that we (NSIDC) determined to be a blocker to reestablish temporal subsetting support.